app/views/admin/shared/_tabnav_order.html.haml

```diff
@@ -4,5 +4,5 @@
     = display_tab "Problem Orders", show_problems_facility_orders_path, action: :show_problems
   - if SettingsHelper.has_review_period? && current_ability.can?(:disputed, Order)
     = display_tab "Disputed", disputed_facility_orders_path, action: :disputed
-  - if current_user.manager_of?(current_facility)
+  - if current_ability.can?(:new, OrderImport)
```

Comment:
TODO: Eliminate the remaining `operator_of?`/`manager_of?` checks in views and controllers